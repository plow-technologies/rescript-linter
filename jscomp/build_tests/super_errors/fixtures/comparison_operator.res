let f = Some(0)

let x = 100 === f
