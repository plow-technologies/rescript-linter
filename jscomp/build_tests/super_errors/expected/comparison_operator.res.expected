
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/comparison_operator.res[0m:[2m3:17[0m

  1 [2mâ”‚[0m let f = Some(0)
  2 [2mâ”‚[0m 
  [1;31m3[0m [2mâ”‚[0m let x = 100 === [1;31mf[0m
  4 [2mâ”‚[0m 

  This has type: [1;31moption<int>[0m
  But it's being compared to something of type: [1;33mint[0m

  You can only compare things of the same type.